pkg/v1/content/content_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package content_test
+
+import (
+	"testing"
+
+	"github.com/google/go-containerregistry/pkg/v1/content"
+)
+
+func TestImageFromContents(t *testing.T) {
```

Comment:
Re: my earlier comment, it might be useful to have a test that asserts that two images from the same content results in the same digest etc.

Reply:
Added